Resources/Textures/Structures/Storage/Crates/piratechest.rsi/meta.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": 1,
+	"license": "CC-BY-SA-3.0",
+	"copyright": "Made by brainfood1183 (Github) for ss14",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+        {
+            "name": "crate",
+            "select": [],
+            "flags": {}
+        },
+        {
+            "name": "crate_door",
+            "select": [],
+            "flags": {}
+        },
+        {
+            "name": "crate_icon",
+            "select": [],
+            "flags": {}
+        },
+        {
+            "name": "welded",
+            "select": [],
+            "flags": {}
+        },
+        {
+            "name": "sparking",            "delays": [[0.1,0.1,0.1,0.1,0.1,0.1]]
+        },
```

Comment:
format delays correctly (should be done just fine by your IDE but if it doesnt work there are a lot of examples around of how it should look)